### PR TITLE
Staging oin

### PR DESCRIPTION
**Pull request by IanHakes-okta**
Promotion of the OIN content (SSO and provisioning guides) migrating them from their home on okta.com/integrate/

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oin-build-provisioning-integration/okta-verification/index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Become Okta Verfied in the OIN
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/oin-build-provisioning-integration/okta-verification/index.md]: Verified

File: packages/@okta/vuepress-site/docs/guides/oin-build-provisioning-integration/overview/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+title: Build a provisioning integration with Okta
+---
+
+Single sign-on is just one aspect of federation. First, an account for the user must be created to grant access to your application. In order to “automate” this process, your application should expose APIs to manage the account lifecycle such as user account creation, profile updates, authorization settings (such as groups or roles), account deactivation, etc.
+
+While many ISVs use proprietary APIs, Okta recommends that you implement your API using the [System for Cross-domain Identity Management](http://www.simplecloud.info) (SCIM) which supports all of the key features needed in provisioning.
+
+For details on how to integrate your application with SCIM, go to Okta’s [SCIM Provisioning Developer Program](/docs/concepts/oin-scim/) for additional guidance. Once you are ready you can submit your app by following the steps below.
+
+Publishing Your SCIM-Based Provisioning Integration
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/oin-build-provisioning-integration/overview/index.md]: Should this be a heading?

File: packages/@okta/vuepress-site/docs/guides/oin-build-provisioning-integration/submit-for-review/index.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+Check your Profile Attributes as follows:
+
+* From the "Admin" section in Okta, open the settings page for your application.
+* In the "Provisioning" tab, scroll to the bottom and click the "Edit Attributes" button in the "User Attributes" section.
+* A "Profile Editor" screen will open, check the following settings:
+  * The "Display name" for the application
+  * The "Description"
+  * In the "Attributes" section, remove all attributes that are not supported by your application.
+
+    This is an important step! Your users will get confused if your application appears to support attributes that are not supported by your SCIM API.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/oin-build-provisioning-integration/submit-for-review/index.md]: This looks like it should be a note

```
> **Note:**
```

File: packages/@okta/vuepress-site/docs/guides/oin-build-provisioning-integration/okta-verification/index.md (hunk 1 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/oin-build-provisioning-integration/okta-verification/index.md]: Fixed - thanks!

**Comment by jakubvul-okta** [comment]
Can we be consistent in the use of "-" instead of "_" in file and directory names?

**Pull request merged**